clientv3/balancer.go

```diff
@@ -222,11 +214,14 @@ func (b *simpleBalancer) Close() error {
 	}
 	b.closed = true
 	close(b.notifyCh)
-	// terminate all waiting Get()s
-	b.pinAddr = ""
-	if len(b.upEps) == 0 {
+	select {
```

Comment:
is this necessary? Isn't Get's ctx canceled around the time Close() is called, so Get exits anyway?